static/js/tippyjs.js

```diff
@@ -158,4 +161,33 @@ export function initialize() {
         hideOnClick: true,
         theme: "light-border",
     });
+
+    // compose box buttons popover shown on mobile widths.
+    delegate("body", {
+        target: ".compose_mobile_button",
+        placement: "top",
+        onShow(instance) {
+            instance.setContent(
```

Comment:
@andersk just FYI -- this is a new style of inserting HTML into the DOM; do we want to use `_html` type variable names with it for your eslint plugin strategy?

Reply:
Oh, yeah. This particular call is fine because it’s easy to see statically that the argument comes directly from a template, but we have other Tippy `setContent` calls that are less clear about the argument being HTML…